content/zh/news/releases/1.18.x/announcing-1.18/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Announcing Istio 1.18.0
```

Comment:
There are also change notes and upgrade notes included in the 1.18 release. I think they should be translated together.

Reply:
https://github.com/istio/istio.io/blob/7ecfdf222e1aa762e772da1755eefe9d51b76d71/content/en/news/releases/1.18.x/announcing-1.18/change-notes/index.md